### ISSUE TITLE
no error for wrong socket

### ISSUE THREAD

**Issue by KES777** [unbit/uwsgi#1159]
running `-s unix:///run/uwsgi/app/site/socket` gives no error

    uwsgi socket 0 bound to TCP address unix:///run/uwsgi/app/site/socket fd 9

**Comment by xrmx** [comment]
--socket should take a unix socket too. Don't know if it does the protocol parsing from the string you passed too. If it's the case then only the message is misleading, s/TCP//.

**Comment by KES777** [comment]
Yes, I do not expect 'TCP' to see here. If it may take a unix socket too.
